### PR TITLE
[export] Fix cond for pass_base

### PR DESCRIPTION
**Pull request by angelayi**
I ported over the code for the inline interpreter incorrectly in the pass base 😅 

Originally the function `make_inline_interpreter` is supposed to take in a fx.Interpreter type but I accidentally passed in an fx.Interpreter object. Also realized while modifying this diff (and comments from Tugsuu) that we don't really need this InlineInterpreter.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #100836

### CODE REVIEW

**Review by tugsbayasgalan — COMMENTED** [review]
Sorry I am not super familiar with ExportPass, could you provide a summary of what you changed and how it fixed the problem?

File: torch/_export/pass_base.py [hunk 1]
```diff
@@ -264,12 +259,11 @@ def _fx(
         args: Tuple[Argument, ...],
         kwargs: Dict[str, Argument],
         meta: NodeMetadata,
-        interpreter: torch.fx.Interpreter,
     ) -> ProxyValue:
         args_data, kwargs_data = pytree.tree_map_only(
             ProxyValue, lambda x: x.data, (args, kwargs)
         )
-        res_data = getattr(interpreter, kind)(target, args_data, kwargs_data)
+        res_data = getattr(InlineInterpreter(), kind)(target, args_data, kwargs_data)
```
Comment (tugsbayasgalan) [inline torch/_export/pass_base.py:266]: Previously it seems like we used to reuse interpreter, but now we are creating a new one each time we invoke fx operator. Is it intended?

Comment (angelayi) [reply torch/_export/pass_base.py]: You're right -- sorry, I don't think we need the InlineInterpreter anymore...will just get rid of it and use the interpreter that we already have 😂

**Review by zhxchen17 — APPROVED** [review]

**Review by tugsbayasgalan — APPROVED** [review]

**Pull request closed without merging**